packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -326,14 +252,37 @@ export default class Tile extends Surface {
       ...labelPatch,
       x: [
         labelPatch.x,
-        this._shouldShowMetadata
+        this._shouldShowBadgeLabel
           ? this.style.animationEntrance
           : this.style.animationExit
       ],
-      ...this._topMetadataTransitions // Badge and Label should animate in with the same values
+      ...this._badgeLabelTransitions
     });
   }
+  /* ------------------------------ Badge & Label ------------------------------ */
 
+  // Badge and Label should animate in with the same values
+  get _shouldShowBadgeLabel() {
+    return this._persistentMetadata || this._isFocusedMode;
+  }
+  get _badgeLabelTransitions() {
+    return {
+      y: [
+        this._shouldShowBadgeLabel ? this.style.paddingY : 0,
+        this.__shouldShowBadgeLabel
+          ? this.style.animationEntrance
+          : this.style.animationExit
+      ],
+      alpha: [
+        this._shouldShowBadgeLabel && !this._isCircleLayout ? 1 : 0.001,
```

Comment:
I'm a little confused by all the alpha logic for Badge and Label. You have this here, but also basically the same thing in the patch for the updateBadge and updateLabel methods.